homeassistant/components/matter/__init__.py

```diff
@@ -218,33 +237,30 @@ async def set_wifi(call: ServiceCall) -> None:
         set_wifi,
         vol.Schema(
             {
-                "network_name": str,
+                "ssid": str,
```

Comment:
If we change the service parameter we need to update `services.yaml`.

Reply:
what do you think ? I think ssid makes more sense to users ?

Reply:
Personally, yes.

We plan to remove these services once we have the frontend in place, right?

Reply:
correct

Reply:
Yes